content/en/docs/concepts/security/index.md

```diff
@@ -617,13 +614,12 @@ services in an Istio Mesh. It features:
     >}}
 
 The above diagram shows the basic Istio authorization architecture. Operators
-specify Istio authorization policies using `.yaml` files. Once deployed, Istio
-saves the policies in the `Istio Config Store`.
+specify Istio authorization policies using `.yaml` files.
 
-Pilot watches for changes to Istio authorization policies. It fetches the
-updated authorization policies if it sees any changes. Pilot distributes Istio
-authorization policies to the Envoy proxies that are co-located with the
-service instances.
+Galley watches for changes to Istio authorization policies. It fetches the
```

Comment:
I think we should remove this paragraph, or move it to the architecture document. In general, we want to be talking about the features and not how the features are implemented.

@rcaballeromx

Reply:
@geeknoid I agree. This information doesn't belong here and we should move it to, at the very least an architecture section, or ideally the Architecture page under reference.